dissect/hypervisor/disk/vmdk.py

```diff
@@ -420,11 +458,16 @@ def parse(cls, vmdk_config):
                 continue
 
             if line.startswith("RW ") or line.startswith("RDONLY ") or line.startswith("NOACCESS "):
-                access_type, size, extent_type, filename = line.split(" ", 3)
-                filename = filename.strip('"')
-                size = int(size)
-                sectors += size
-                extents.append([access_type, size, extent_type, filename])
+                # Extent descriptors can have up to seven values according to libvmdk documentation.
+                parts = line.split(" ", maxsplit=6)
+
+                if len(parts) < 3:
+                    log.error("Unexpected ExtentDescriptor format in vmdk config: %s", line)
```

Comment:
```suggestion
                    log.warning("Unexpected ExtentDescriptor format in vmdk config: %s, ignoring", line)
```
Error would imply something is breaking, but we just ignore it.